index.html

```diff
@@ -1,75 +1,395 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Proiect HTML+CSS</title>
-    <style>
-        body {
-            margin: 0;
-            min-height: 100vh;
-        }
-    </style>
-</head>
-<body>
-    <h1>Cornel</h1>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="stylesheet.css" />
+    <script src="fairyDustCursor.js"></script>
```

Comment:
Până la proiectul viitor, îți dau o provocare personală: Modifică scriptul din `fairyDustCursor.js` astfel încât să folosească animații CSS.

Reply:
Voi incerca asta, sper ca data viitoare sa am timpul necesar sa creez tot ce imi propun. Acum chiar am fost data peste cap sa realizez proiectul inainte de a pleca in vacanta.